Leibit.Core/Data/sued/MLI.xml

```diff
@@ -0,0 +1,221 @@
+    </track>
+    <track name="6">
+      <block name="62B6"/>
+    </track>
+  </station>
+
+  <station name="Röthenbach" short="MRTA" refNr="63" scheduleFile="g63_____.abf" localOrderFile="bf63____.abf">
+    <track name="1">
+      <block name="63G1"/>
+      <alternative track="2"/>
+      <alternative track="3"/>
+    </track>
+    <track name="2">
+      <block name="63G2"/>
+    </track>
+    <track name="3">
+      <block name="63G3"/>
+    </track>
+    <track name="51">
+      <block name="63G51"/>
+      <alternative track="2"/>
+      <alternative track="3"/>
+    </track>
+  </station>
+
+  <station name="Hergatz" short="MHGZ" refNr="64" scheduleFile="g64_____.abf" localOrderFile="bf64____.abf">
+    <track name="1">
+      <block name="64G1"/>
+    </track>
+    <track name="2">
+      <block name="64G2"/>
+    </track>
+    <track name="3">
+      <block name="64G3"/>
+    </track>
+  </station>
+
+  <station name="Lindau-Reutin" short="MLIR" refNr="68" scheduleFile="g68_____.abf" localOrderFile="bf68____.abf">
+    <track name="1">
+      <block name="68G1"/>
+    </track>
+    <track name="2">
+      <block name="68G2"/>
+    </track>
+    <track name="3">
+      <block name="68G3"/>
+    </track>
+    <track name="4">
+      <block name="68G4"/>
+    </track>
+    <track name="5">
+      <block name="68G5"/>
+    </track>
+    <track name="6">
+      <block name="68G6"/>
+    </track>
+    <track name="9">
+      <block name="68G9"/>
+    </track>
+  </station>
+
+  <station name="Lindau Hbf" short="MLI" refNr="67" scheduleFile="g67_____.abf" localOrderFile="bf67____.abf">
+    <track name="1">
+      <block name="67G1"/>
+    </track>
+    <track name="2">
+      <block name="67G2"/>
+    </track>
+    <track name="3">
+      <block name="67G3"/>
+    </track>
+    <track name="4">
+      <block name="67G4"/>
+    </track>
+    <track name="5">
+      <block name="67G5"/>
+    </track>
+    <track name="6">
+      <block name="67G6"/>
+    </track>
+    <track name="7">
+      <block name="67G7"/>
+    </track>
+    <track name="8">
+      <block name="67G8"/>
+    </track>
+    <track name="9">
+      <block name="67G9"/>
+    </track>
+  </station>
+
+  <station name="Nonnenhorn" short="MNHN" refNr="73" scheduleFile="g73_____.abf" localOrderFile="bf73____.abf">
+    <track name="1">
+      <block name="73G1"/>
+    </track>
+    <track name="2">
+      <block name="73G2"/>
+    </track>
+  </station>
+
+  <station name="Wasserburg" short="MWBG" refNr="72" scheduleFile="g72_____.abf">
+    <track name="1">
+      <block name="72B1"/>
+    </track>
+  </station>
+
+  <station name="Enzisweiler" short="MENW" refNr="71" scheduleFile="g71_____.abf" localOrderFile="bf71____.abf">
+    <track name="1">
+      <block name="71G1"/>
+    </track>
+    <track name="2">
+      <block name="71G2"/>
+    </track>
+  </station>
+
+  <station name="Lindau-Aeschach" short="MLIA" refNr="66" scheduleFile="g71_____.abf">
```

Comment:
Das ScheduleFile passt nicht. Ist das gleiche wie in Enzisweiler. Vermutlich müsste es "b71_____.abf" sein. Dann müsstest du auch die Gleisnamen nochmal anpassen, damit die genauso heißen wie in der Datei. Die Zuordnungen der Blöcke und Richtungen passt aber soweit.

Reply:
Ok, ich glaub da hab ich gerade noch Verständnis schwierigkeiten.
Das Attribut "Track.Name" beziehe ich aus der Fahrplandatei?
Das Attribut "Block.Name" entweder aus dem Stellwerk oder besser aus der ESTWOnline Datei.
Wie findet die richtige Zuordnung zueinander, gerade in Aeschach statt? In die eine Seite ist es ja recht klar, da führt jeweils ein Block in die drei Gleise. aber auf der anderen Seite kann ja ein Block in fast jedes der drei Gleise führen.

Reply:
Genau, Track.Name entspricht der Angabe in der Fahrplandatei (.abf) und Block.Name ist die Bezeichnung des ZN-Felds aus der ESTWonline-Datei (.dat), wobei die in 90% der Fälle mit den Bezeichnungen in der Sim übereinstimmt. Ausnahmen sind mir nur von Spiegelfeldern bekannt (mit dem "S" im Blocknamen) und von Betriebsstellen mit einer Bereichskennziffer < 10 (z.B. Burgsinn Bbf, das Gleis 1 heißt dort in ESTWsim "6G001" und in der ESTWonline-Datei "06G001".

Die Zuordnung von Gleisen zu Blöcken ist gerade bei Üst'en und Abzw'en nicht immer eindeutig möglich. Genau dafür ist das Attribut isPlatform="false" gedacht. Das sorgt dafür, dass es egal ist, auf welchem Gleis einer Betriebsstelle ein Zug fährt. In diesem Fall wird auch nie ein abweichendes Gleis gesetzt, weil es ja wie gesagt eh nicht zweifelsfrei geht.
Als Referenz kannst du ja mal schauen, wie das in Hamburg mit den ganzen Abzweigen zwischen Pz- und Gz-Gleisen gelöst ist (z.B. Ericus oder Veddel). Oder auch der Abzw. Nantenbach, den ich just gestern gebaut habe (Branch "nlo").

Reply:
Habe ich mal angepasst. Ich hab jetzt wie bei dir in Nantenbach auch die Blöcke den tatsächlichen Gleisen zugeordnet.